configure

```diff
@@ -622,7 +634,7 @@ sub write_platform_macros {
       $plat = $pi->{'aceplatform'};
       $plat =~ s/\$COMP/($opts{'compiler'} =~ m!CC!) ? 'sunc++' : 'g++'/e;
       $plat =~ s/\$NONSTDCOMP/($opts{'compiler'} =~ m!clang!) ? 'clang' : ''/e;
-      $plat =~ s/_$//e; # Trailing _
+      $plat =~ s/_//; # Trailing _
```

Comment:
Oh yes, this was a temporary change due to a warning.  Reverted back to
_$...

On Mon, Jul 6, 2015 at 2:48 PM, Adam Mitz <notifications@github.com> wrote:

> In configure
> <https://github.com/objectcomputing/OpenDDS/pull/91#discussion_r33974391>:
>
> > @@ -622,7 +634,7 @@ sub write_platform_macros {
> >        $plat = $pi->{'aceplatform'};
> >        $plat =~ s/\$COMP/($opts{'compiler'} =~ m!CC!) ? 'sunc++' : 'g++'/e;
> >        $plat =~ s/\$NONSTDCOMP/($opts{'compiler'} =~ m!clang!) ? 'clang' : ''/e;
> > -      $plat =~ s/_$//e; # Trailing _
> > +      $plat =~ s/_//; # Trailing _
>
> If It is only trailing _ (as comment indicates), use $ in regex.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/objectcomputing/OpenDDS/pull/91/files#r33974391>.
>